recipes/polly/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "16.0.4" %}
```

Comment:
Should we start with 15 here (to align with the current stack) and then upgrade to 16 in the feedstock?